programs/bpf_loader/src/syscalls/cpi.rs

```diff
@@ -763,7 +930,7 @@ where
         } else if let Some(caller_account_index) =
             account_info_keys.iter().position(|key| *key == account_key)
         {
-            let original_data_len = accounts_metadata
+            let serialied_metadata = accounts_metadata
```

Comment:
typo: `serialized_metadata` (same as this: https://github.com/solana-labs/solana/pull/31986/files#r1279270148)